src/actionbar/index.css

```diff
@@ -0,0 +1,46 @@
+@import '../commons/index.css';
+
+:root {
+  --spectrum-actionbar-height: var(--spectrum-global-dimension-size-600);
+  --spectrum-actionbar-padding-x: var(--spectrum-global-dimension-size-200);
+  --spectrum-actionbar-margin-x: var(--spectrum-global-dimension-size-200);
+  --spectrum-actionbar-offset-y: var(--spectrum-global-dimension-size-200);
+
+  --spectrum-actionbar-min-width: 280px;
+  --spectrum-actionbar-max-width: 1280px;
+}
+
+/* topdoc
+{{ actionbar/actionbar.yml }}
+*/
+.spectrum-Actionbar {
+  align-items: center;
+  flex-direction: initial;
+  justify-content: space-between;
+
+  height: var(--spectrum-actionbar-height);
+  padding: 0 var(--spectrum-actionbar-padding-x);
+  min-width: var(--spectrum-actionbar-min-width);
+  max-width: var(--spectrum-actionbar-max-width);
+  width: 100%;
+}
+
+/* topdoc
+{{ actionbar/table-actionbar.yml }}
+*/
+.spectrum-Actionbar-tableContainer {
```

Comment:
Hmmm, is there any way to do this without a parent element? Could we put `position: sticky` on a modifier class that we add, something like`spectrum-ActionBar spectrum-ActionBar--sticky` that makes it sticky and sets the transform?

Reply:
The container element is for centering the action bar and enforcing the left and right margin. I will give it  an another try. 

I totally agree with you on `position: sticky`, it should not be hard coded. `position: fixed` will be another possible value for a super long table which longer than the current viewport. Any thoughts that we can make it more flexible?

Reply:
@jianliao ok, let's see if we can pull it off without the container. Thanks!

Reply:
@lazd I tried, but seems like container element is a must for supporting both `fixed` and `sticky` strategies. When applied `position: fixed`, action bar content lost the connection with its parent element e.g. table body. A container element is for simulating its parent to set a width on and make centered easy.

I generalized it by renaming `.spectrum-Actionbar-tableContainer` to `.spectrum-Actionbar-container`. Here is a [live demo](http://angular-spectrum.corp.adobe.com/component-viewer/table-basic) for `position: fixed`.